src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -182,6 +186,25 @@ export class DictionaryConcepts extends Component {
     return newConcepts;
   }
 
+  handleChange = (event) => {
+    const {
+      target: { value },
+    } = event;
+    this.setState(() => ({ madeSearch: true, searchInput: value }));
+    const { madeSearch } = this.state;
+    if (madeSearch && value.length === 0) {
```

Comment:
Why are you making the search when it was already done? Am basing on the variable named madeSearch

Reply:
@dkayiwa the boolean madeSearch was supposed to check if the user had voluntarily typed (made a search) in the search input. On second thought it's not necessary because an event is in charge of providing the value and an if check on that is enough. Let me take that boolean out.